examples/Tile.main.tsx

```diff
@@ -13,12 +13,12 @@ export default () => {
       <Tile.ThumbnailArea>
         <Tile.ThumbnailPicture url='https://itwinplatformcdn.azureedge.net/iTwinUI/stadium.png' />
         <Tile.TypeIndicator>
-          <Tile.IconButton>
+          <Tile.IconButton aria-label='Favorite'>
```

Comment:
try using the `label` prop instead of `aria-label`. it will add a tooltip for sighted users

Reply:
Sounds good, changed